src/Tribe/Validate.php

```diff
@@ -239,7 +239,10 @@ public function slug() {
 
 			// esc_url_raw does the work of validating chars, but returns the checked string with a
 			// prepended URL protocol; so let's use strpos to match the values.
-			if ( false !== strpos( $maybe_valid_value, $this->value ) ) {
+			if (
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: d0377b92f723e89a630b0153dfb3db59</sub>